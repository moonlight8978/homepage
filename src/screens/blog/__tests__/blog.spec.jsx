import React from 'react'
import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'

import BlogScreen from '../blog'

test('renders post list', () => {
  const { container } = render(
    <MemoryRouter>
      <BlogScreen />
    </MemoryRouter>
  )
  expect(container.firstChild).toMatchSnapshot()
})
