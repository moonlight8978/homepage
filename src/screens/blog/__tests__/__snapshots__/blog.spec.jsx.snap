// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders post list 1`] = `
<div
  class="postList"
>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
  <div
    class="postItem"
  >
    <div
      class="post-list-item"
    >
      <a
        class="link"
        href="/blog/mysql-optimize-where"
      >
        <h2
          class="title"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </h2>
      </a>
      <span
        class="creation"
      >
        <span>
          January 13, 2020
        </span>
        <span>
           by _MoonLight_
        </span>
      </span>
      <div
        class="tags"
      >
        <div
          class="tag"
        >
          Javascript
        </div>
        <div
          class="tag"
        >
          Frontend
        </div>
        <div
          class="tag"
        >
          React
        </div>
      </div>
      <div
        class="paragraph"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in …
      </div>
    </div>
  </div>
</div>
`;
